shotgun_api3/shotgun.py

```diff
@@ -2164,6 +2176,27 @@ def _make_call(self, verb, path, body, headers):
             attempt += 1
             try:
                 return self._http_request(verb, path, body, req_headers)
+            except SSLHandshakeError, e:
+                # Not a SHA-2 error or SHA-2 error we want to enforce. SHA-2 errors look like: 
```

Comment:
It would be good to say what is going on here or at least refer to the warning message below in the comment.  So if somebody is just reading the comments they see what the code will do.